doc/dune-files.rst

```diff
@@ -698,6 +702,45 @@ With this dune file, running dune as follow will replace the
 
     $ dune build @runtest --auto-promote
 
+.. _tests-stanza:
+
+tests
+-----
+
+The ``tests`` stanza allows one to easily define multiple tests. For example we
+can define two tests at once with:
+
+.. code:: scheme
+
+   (tests
+    (names mytest expect_test)
+    <optional fields>)
+
+This will define an executable named ``mytest.exe`` that will be executed as
+part of the ``runtest`` alias. If the directory also contains an
+``expect_test.expected`` file, then ``expect_test`` will be used to define an
+expect test. That is, the test will be executed and its output will be compared
+to ``expect_test.expected``.
+
+The optional fields that are supported a subset of alias and executables fields.
```

Comment:
There is a word missing in this sentence

Reply:
Thanks. Fixed.